HW1/P4/P4.py

```diff
@@ -0,0 +1,35 @@
+# import findspark
+# findspark.init()
+import pyspark
+import numpy as np
+
+from P4_bfs import *
+
+
+# We lode the data and create the graph
+
+superheroes = sc.textFile('./P4/source.csv')
+
+ByArticle = superheroes.map(lambda line: (line.split('","')[1][:-1], line.split('","')[0][1:]))
```

Comment:
You could make this more efficient by only calling line.split() once, instead of for both elements in the tuple